### PR TITLE
Tici camerad

### PR DESCRIPTION
**Pull request by robbederks**

### CODE REVIEW

**Comment by geohot** [comment]
Junk files @ and 1. This is still a huge PR!

File: selfdrive/camerad/main.cc [hunk 1]
```diff
@@ -1211,13 +1546,18 @@ void party(VisionState *s) {
                        processing_thread, s);
   assert(err == 0);
 
-#if !defined(QCOM2) && !defined(__APPLE__)
-  // TODO: fix front camera on qcom2
+#if !defined(__APPLE__)
```
Comment (adeebshihadeh) [inline selfdrive/camerad/main.cc:1549]:
```suggestion
#ifndef __APPLE__
```

File: selfdrive/camerad/SConscript [hunk 2]
```diff
@@ -11,6 +11,10 @@ if arch == "aarch64":
 elif arch == "larch64":
   libs += []
   cameras = ['cameras/camera_qcom2.c']
+  # no screen
+  env = env.Clone()
```
Comment (adeebshihadeh) [inline selfdrive/camerad/SConscript:15]:
```suggestion
  #env = env.Clone()
```

File: selfdrive/camerad/main.cc [hunk 3]
```diff
@@ -1157,25 +1473,42 @@ void init_buffers(VisionState *s) {
...
   for (int i=0; i<FRAME_BUF_COUNT; i++) {
     visionbuf_free(&s->camera_bufs[i]);
-    visionbuf_free(&s->front_camera_bufs[i]);
+#ifdef QCOM
     visionbuf_free(&s->focus_bufs[i]);
     visionbuf_free(&s->stats_bufs[i]);
+#endif
   }
 
+  for (int i=0; i<FRAME_BUF_COUNT; i++) {
```
Comment (adeebshihadeh) [inline selfdrive/camerad/main.cc:1495]: this is the same as the for loop directly above

**Pull request merged**